library/appointments.inc.php

```diff
@@ -570,4 +606,72 @@ function fetchAppointmentCategories()
             . " FROM openemr_postcalendar_categories WHERE pc_recurrtype=0 and pc_cattype=0 ORDER BY category";    
      return sqlStatement($catSQL);
 }
+
+function interpretFreq($freq){
+	global $REPEAT_FREQ;
+	return $REPEAT_FREQ[$freq];
+}
+
+function interpretFreqType($type){
+	global $REPEAT_FREQ_TYPE;
+	return $REPEAT_FREQ_TYPE[$type];
+}
+
+function interpretOnNum($on_num){
+	global $REPEAT_ON_NUM;
+	return $REPEAT_ON_NUM[$on_num];
+}
+
+function interpretOnDay($on_day){
+	global $REPEAT_ON_DAY;
+	return $REPEAT_ON_DAY[$on_day];
+}
+
+function interpretRecurrence($recurr_freq, $recurr_type){
```

Comment:
Couldn't you just bring in the 4 above globals here and then use the original arrays rather than create the 4 above work around functions?